src/style.css

```diff
@@ -1,7 +1,24 @@
-.hello {
-    color: rgb(9, 9, 206);
-  }
+*,
+*::after,
+*::before {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
 
-  body {
-    background-color: rgb(223, 195, 160);
-}
\ No newline at end of file
+body {
+  background-color: rgb(246, 246, 246);
+}
+
+#toDoList {
+  display: flex;
+  flex-direction: column;
+  width: 25%;
+  height: auto;
+  margin: 300px auto auto auto;
+  background-color: white;
+}
+
+::placeholder {
+  font-style: italic;
+}
```

Comment:
- [ ] Kindly style your todo list container to look like the guiding template. Please take a look at this point from the project requirements.
- Create a style.css and set rules for the To Do List. CSS must be loaded by Webpack Style/CSS Loader. Your list should be a clone of the part of the minimalist project captured in the video below.

![project-req](https://user-images.githubusercontent.com/26597432/201221076-2d93d104-93d3-4b85-b2cf-8c08025e0c79.png)


### Your Design

![your-page](https://user-images.githubusercontent.com/26597432/201221178-236453d0-0b3c-4e70-b9bc-6a5aa75cf9ce.png)